contents/blog/sessions-refactoring.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2021-09-21
+title: (TBD) Sessions refactoring
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: paolodamico
+featuredImage: ../images/blog/titles.png
+featuredImageType: full
+---
+
+Two main session analytics use cases:
+1. Run session-based behavioral analytics. Useful for products with certain types of engagement where interactions over time are important and natural. Examples: Netflix, Slack, Notion.
+2. Explore, debug, deep dive to answer _why_ users are behaving a certain way.
+
+
+The first case we mostly don't support today. The only session-based analysis you can do is session duration distribution and even then it has the fundamental problem of how sessions are computed.
+
+For the second case, sessions is not the best way to solve the problem. Further, all the work we're doing around Paths and Quant Analysis will actually solve for this use case and provide significantly more value. In particular, you'll get the same useful qualitative information, but the quantitative overlay will greatly reduce bias (and effort).
+
+
+Here's therefore an ambitious proposal.
+
+### Actions
+
+- Getting rid of the sessions page altogether. It causes a ton of confusion and provides limited value. In numerical terms, less than [3%](https://app.posthog.com/i/ifiUioRu) of pageviews in the app in the last month were on the sessions page for users with no session recording enabled (proxy metric).
+  - Introduce a session recording list page instead. This page should mostly be around finding specific recordings you want to watch (e.g. customer support, recordings with exceptions, ...). _Funnels will be the starting spot for the other use cases._
+  - On the Person page, we'll include recordings first and events as a secondary tab.
+- Getting rid of the "Sessions" insight. The functionality is quite limited (only a time distribution with limited visualization) and confusing (e.g. the events/actions that compose a session). Further, only [1.5%](https://app.posthog.com/i/y_iUoLva) of insights analyzed in the last month were on "Sessions".
```

Comment:
@paolodamico I've been putting some thought into this one, and I wonder if we leave the "sessions tab" under insights. I feel like it's pretty useful information (trends in avg. session length etc.), and I don't think it's confusing enough to be harmful. Thoughts?

Reply:
My thinking is that it leads to the wrong conclusion that we support session-based analytics and particularly the way we compute sessions might be misguided for certain products. We could remove it (FFed) and give the option to users to get it back, this way they can also provide context as to why it would be useful for them.

Reply:
I'm a bit vary on removing this. I agree with all the reasoning regarding session based analytics, however this is the only place currently a lot of users have to answer a simple question: "how long do users stay on my site"? Even if the info is off by a bit, it's still better than nothing for some users.

Reply:
Added a note we're still testing this.